DisplayInfo/DisplayInfo.h

```diff
@@ -16,12 +16,17 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
- 
+
 #pragma once
 
 #include "Module.h"
 #include <interfaces/IDisplayInfo.h>
 #include <interfaces/json/JsonData_DisplayInfo.h>
+#include <vector>
+
+#if defined(USE_IARM)
```

Comment:
I thought that the aim was to remove all DBus references. DBUS caters for a blackboard pattern architecture which, I think, in a closed system as ours is makes the dependencies hard to resolve.
The aim of the Thunder architecture is to create self-contained plugins that can do what they are supposed to do without delegating it to new components.
By not delegating, the dependencies are clear and resource consumption is reduced.

Reply:
Hi Pierre,
Removing all DBUS references is indeed the final goal. In fact, we looked at possible ways to avoid making IARM calls from the plugin. However, we also wanted to make use of deviceSettings library calls to obtain the device properties. As it turns out, devicesettings library on the Arris Xi6 works in a "multi-app" mode which has a tight coupling with IARM (We are required to initialize the IARM bus and connect to it prior to making any devicesettings calls
![MultiAppMode](https://user-images.githubusercontent.com/55712051/87489854-b8a26800-c661-11ea-88a2-34b3bffc48df.jpg)
). Due to this tight dependency, it was decided to keep the IARM calls in the plugin for now. We will be removing the IARM calls in the next phase.